private-templates-service/client/src/components/Designer/styled.tsx

```diff
@@ -4,6 +4,7 @@ export const OuterDesignerWrapper = styled.main`
   display: flex;
   flex-direction: column;
   height: 100%;
+  overflow-behaviour: contain;
```

Comment:
I think it's a typo: `overscroll-behavior` - it should be

Reply:
my bad, fixed